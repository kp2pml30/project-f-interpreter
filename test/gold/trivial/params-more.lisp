(func foo (a) (return 10))
(foo 1 2)
