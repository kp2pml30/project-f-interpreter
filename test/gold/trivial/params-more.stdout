Error: wrong arguments number got 2, expected 1 for <func {"overallPos":0,"line":1,"offset":0}>; stack:
	{"overallPos":27,"line":2,"offset":0}